crates/rattler_repodata_gateway/Cargo.toml

```diff
@@ -13,6 +13,7 @@ readme.workspace = true
 [dependencies]
 async-compression = { workspace = true, features = ["gzip", "tokio", "bzip2", "zstd"] }
 blake2 = { workspace = true }
+bytes = { workspae = true, optional = true }
```

Comment:
```suggestion
bytes = { workspace = true, optional = true }
```

Reply:
Thanks! Weird that the CI didn't catch it